lib/easypost/services/referral_customer.rb

```diff
@@ -44,7 +44,8 @@ def add_credit_card(referral_api_key, number, expiration_month, expiration_year,
         easypost_stripe_api_key,
       )
     rescue StandardError
-      raise EasyPost::Error.new('Could not send card details to Stripe, please try again later.')
+      msg = EasyPost::Constants::STRIPE_CARD_CREATE_FAILED
+      raise EasyPost::Errors::ExternalApiError.new(msg)
```

Comment:
suggestion: inline the constant, there is no need to reassign it
```suggestion
      raise EasyPost::Errors::ExternalApiError.new(
          EasyPost::Constants::STRIPE_CARD_CREATE_FAILED
      )
```